static/js/narrow.js

```diff
@@ -156,7 +157,9 @@ function update_narrow_title(filter) {
 }
 
 export function hide_mark_as_read_turned_off_banner() {
-    $("#mark_as_read_turned_off_banner").hide();
+    // Use visibility instead of hide() to prevent messages on the screen from
+    // shifting vertically.
+    $("#mark_as_read_turned_off_banner").css({visibility: "hidden"});
```

Comment:
It feels wrong for this function to be in a different file from `unread_ui.reset_mark_as_read_turned_off_banner`. Maybe add a prep commit moving this function to `unread_ui.js`?

Reply:
Among other reasons, the comment about visibility would feel a lot better if this were next to its inverse.

Reply:
💯 great point, thanks